libs/commons/Logs_helpers.ml

```diff
@@ -1,23 +1,50 @@
+(* ansii escape sequences for colored output, depending on log level *)
+let color level =
+  match level with
+  | Logs.Warning -> Some "33" (*yellow*)
+  | Logs.Error -> Some "31" (*red*)
+  | _else -> None
+
+(* print an ansi escape sequence *)
+let pp_sgr ppf style =
+  Format.pp_print_as ppf 0 "\027[";
```

Comment:
there's an OCaml library ANSITerminal that we could use maybe here (we already use it in libs/commons/Console.ml,
so no need to add it to the dependencies). Or maybe just add a comment explaining why we don't use
ANSITerminal lib here (not worth it).